cli/src/auth.rs

```diff
@@ -183,15 +183,18 @@ where
 	T: Serialize + ?Sized,
 {
 	let dec = serde_json::to_string(value).expect("expected to serialize");
-	encrypt(&dec)
+	if std::env::var("VSCODE_CLI_DISABLE_ENCRYPT").is_ok() {
```

Comment:
```suggestion
	if std::env::var("VSCODE_CLI_DISABLE_KEYCHAIN_ENCRYPT").is_ok() {
```

Reply:
doesn't this also disable encryption if a file is used instead of the keychain?

Reply:
In this I use "keychain" as a general purpose term for the place where secrets are stored; each platform has their own name for it anyway (keychain on macOS, secret store on Linux, credential manager on Windows). We already have a similar variable VSCODE_CLI_USE_FILE_KEYCHAIN.